index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+    <header>
+    <div class="pure-nav-list">
+            <h4 class="important"><a href="#">Ayodele</a></h4>
+            <nav>
+                <div class="menu1">
+                    <div class="menu"></div>
+                    <div class="menu"></div>
+                    <div class="menu"></div>
+                </div>
+            </nav>
+            </div>
+            
+        </header>
+
+        <main>
+    <section class="section-headline">
+    <div class="main-text Developer">
+    <div class="introduction">
+    <h1>Hey There. <br> I’m Ayodele</h1>
+    <h3>I’am a Software Developer</h3>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    
+    </div>
+
+    <ul class=" social-media social-media-icons mv-social-media mv-social-media-icons">
+        <li class="social-1"><a href="#"><i class="fab fa-github fa-lg"></i></a></li>
+        <li class="social-2"><a href="#"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
+        <li class="social-3"><a href="#"><i class="fab fa-angellist fa-lg"></i></a></li>
+        <li class="social-4"><a href="#"><i class="fab fa-twitter fa-lg"></i></a></li>
+        <li class="social-5"><a href="#"><i class="fab fa-medium-m"></i></a></li>
+        </ul>
+
+    </div>
+    
+    <div class="hero-image">
+        <img src="images/background-mobile.svg" alt="" />
```

Comment:
### OPTIONAL
- [ ] The section has the proper background image (decorations are always background images, not img tags)